t/091-coroutine.t

```diff
@@ -483,8 +483,8 @@ done
 GET /lua
 --- response_body
 hello
---- error_log eval
-["stack traceback:", "coroutine 0:", "coroutine 1:", "coroutine 2:"]
+--- no_error_log
+[error]
```

Comment:
Hmm, the indentation of this test case is to test the Lua land backtrace. After your change, we no longer have any tests designed for this purpose.

I'm not sure if silencing the error logging is a good idea. Maybe we can make it a lower log level like "warn" or even "info"?